### DISCUSSION TITLE
How to create vertical tabs?

### DISCUSSION THREAD

**Discussion by qarmin** [slint-ui/slint#3686] [Q&A]
For now I see only horizontal tabs

and TabWidget not have any docs mentioning about this feature

https://slint.dev/releases/1.2.2/docs/slint/src/language/widgets/tabwidget

**Comment by ghost** [comment]
Hi @qarmin,

thank you for the question. It's not possible at the moment but I think that is something we should add in the future. 

What you could do now to do archive it is to create a custom one there you can find an example: https://slint.dev/releases/1.2.2/docs/slint/src/recipes/recipes#custom-tabs
